tests/test_teststats.py

```diff
@@ -12,7 +12,7 @@ def test_qmu(caplog):
     par_bounds = model.config.suggested_bounds()
 
     with caplog.at_level(logging.WARNING, "pyhf.infer.test_statistics"):
-        pyhf.infer.test_statistics.qmu(mu, data, model, init_pars, par_bounds)
+        pyhf.infer.test_statistics.qmu(mu, data, model, init_pars, par_bounds, [])
```

Comment:
```suggestion
        pyhf.infer.test_statistics.qmu(mu, data, model, init_pars, par_bounds, fixed_vals)
```

To try and keep this in a similar style as the rest of the function call, and also readable, what about adding a

```python
fixed_vals = []
```
just below `par_bounds = model.config.suggested_bounds()` above and then using it here and then doing the same for the tests below?